server/game/cards/08.5-TFM/TheSongOfTheSeven.js

```diff
@@ -0,0 +1,38 @@
+const DrawCard = require('../../drawcard.js');
+
+class TheSongOfTheSeven extends DrawCard {
+    setupCardAbilities(ability) {
+        this.reaction({
+            max: ability.limit.perRound(1),
+            when: {
+                onDominanceDetermined: event =>
+                    this.controller !== event.winner &&
+                    event.winner.faction.power >= 2 &&
+                    this.hasTheSeven()
+            },
+            target: {
+                cardCondition: card =>
+                    card.location === 'play area' &&
+                    card.controller === this.controller &&
+                    card.getType() === 'character' &&
+                    card.hasTrait('The Seven')
+            },
+            handler: context => {
+                let otherPlayer = context.event.winner;
+                let power = 2;
```

Comment:
This needs to be 1 if ```otherPlayer.faction.power === 1``` and 2 if more than that. movePower guards against not being able to move more power than actually possible, but setting this variable incorrectly would mess up the message.